backend/cn/testGeneration.mli

```diff
@@ -0,0 +1,12 @@
+module CF = Cerb_frontend
+
+type test_framework = GTest
+
+val main
+  :  string
+  -> string
+  -> int
```

Comment:
Can you please use named parameters for the two strings (and maybe the int)?
Can you also add a doc comment for this?